Support/Testing/Travis/install.py

```diff
@@ -15,17 +15,17 @@
 dist_packages = []
 pip_packages = []
 
-linux_packages = { 'Linux-ARM':     'g++-4.8-arm-linux-gnueabihf',
-                   'Linux-X86':     'g++-5-multilib',
-                   'Linux-X86_64':  'g++-5' }
+gcc_package_versions = { 'Linux-ARM':     '-4.8-arm-linux-gnueabihf',
+                         'Linux-X86':     '-5-multilib',
+                         'Linux-X86_64':  '-5' }
 
 android_toolchains = { 'Android-ARM':       'arm',
                        'Android-ARM64':     'arm64',
                        'Android-X86':       'x86',
                        'Android-X86_64':    'x86_64' }
 
-tizen_packages = { 'Tizen-ARM': linux_packages['Linux-ARM'],
-                   'Tizen-X86': linux_packages['Linux-X86'] }
+tizen_packages = { 'Tizen-ARM': 'g++' + gcc_package_versions['Linux-ARM'],
```

Comment:
This means we will only install g++-XXX during the Tizen build. It's ok for now, but it means if we try running tests on Tizen binaries, it won't work.